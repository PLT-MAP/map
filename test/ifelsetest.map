func main() {  
	if (10<7){   
		cost= 2;
	}
	else{
		print("success");
	}
}
