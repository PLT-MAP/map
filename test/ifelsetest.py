import networkx
import sys
def main():
    if (10.0<7.0):
            cost = 2.0
        
        else:
        print "success"
    
if __name__ == '__main__': 
	main()
