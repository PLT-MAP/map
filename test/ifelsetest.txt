def main():
	if (10.0<7.0):
		cost=2.0
	else:
		print ("success")
	
