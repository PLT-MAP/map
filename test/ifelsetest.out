import networkx as nx
import sys
def main():
    if (10.0<7.0):
        if (10.0<7.0):
            print("hi")
        cost = 2.0
    else:
        print("success")
if __name__ == '__main__': 
	try:
		main()
	except:
		print'Error:',sys.exc_info()[0]
		print'Resolve error before running again!'